Formula/servefiles.rb

```diff
@@ -0,0 +1,39 @@
+class Servefiles < Formula
+  include Language::Python::Virtualenv
+  desc "Serve local files to the Nintendo 3DS via the FBI remote installer"
+  homepage "https://gbatemp.net/threads/release-fbi-open-source-cia-installer.386433/"
+  url "https://github.com/Steveice10/FBI/archive/2.4.12.tar.gz"
+  sha256 "aadbc150fee385aa8c0d8ef2b1ab1deba3d5bbf3b65fd3497be565b1b2a380d9"
+  head "https://github.com/Steveice10/FBI.git"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  patch do
```

Comment:
patch is already merged to master so either this needs to move into a `stable do` block, or `head` needs to be removed or we need a new release